ui/src/components/SampleView/SampleImage.jsx

```diff
@@ -528,58 +528,36 @@ export default class SampleImage extends React.Component {
   wheel(e) {
     e.preventDefault();
     e.stopPropagation();
-    const { sampleViewActions, motorSteps, hardwareObjects } = this.props;
-    const { updateMotorPosition } = sampleViewActions;
+    const { hardwareObjects, uiproperties, setAttribute } = this.props;
+    const { components } = uiproperties.sample_view;
     const keyPressed = this._keyPressed;
 
-    const phi = hardwareObjects['diffractometer.phi'];
-    const focus = hardwareObjects['diffractometer.focus'];
-    const zoom = hardwareObjects['diffractometer.zoom'];
-
-    if (keyPressed === 'r' && phi.state === HW_STATE.READY) {
-      // then we rotate phi axis by the step size defined in its box
-      if (e.deltaX > 0 || e.deltaY > 0) {
```

Comment:
I'm no longer checking for `deltaX` here, as this assumes a mouse/pad that supports horizontal scroll, and I feel one would not instinctively scroll horizontally to perform this interaction anyway.